### PR TITLE
Feature/traffic switch

### PR DESCRIPTION
**Pull request by patduin**
### :pencil: Description
- 

### :link: Related Issues
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
...
+| primary_metastore_host_read_only             | Optional Primary Hive Metastore READ ONLY hostname configured in Waggle Dance.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | string |                  ``                    |    no    |
```
Comment (jmnunezizu) [inline README.md:42]: `primary_metastore_read_only_host` and `primary_metastore_read_only_port` reads better.

File: variables.tf [hunk 2]
```diff
@@ -199,6 +199,18 @@ variable "primary_metastore_port" {
   default     = "9083"
 }
 
+variable "primary_metastore_host_read_only" {
+  description = "Optional Primary Hive Metastore READ ONLY hostname configured in Waggle Dance."
```
Comment (jmnunezizu) [inline variables.tf:203]: `Primary Hive Metastore READ ONLY hostname configured in Waggle Dance. Optional.` reads better.

File: variables.tf [hunk 3]
```diff
@@ -199,6 +199,18 @@ variable "primary_metastore_port" {
   default     = "9083"
 }
 
+variable "primary_metastore_host_read_only" {
```
Comment (jmnunezizu) [inline variables.tf:202]: You should update the `README.md` and include them there as well.

**Review by jmnunezizu — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 3 above)
Comment (patduin) [reply variables.tf]: done

File: README.md (hunk 1 above)
Comment (patduin) [reply README.md]: done

**Review by githubjianli — APPROVED** [review]

**Review by mroark1m — APPROVED** [review]

**Pull request merged**